include/benchmark/benchmark_api.h

```diff
@@ -502,11 +508,31 @@ class Benchmark {
   // REQUIRES: The function passed to the constructor must accept arg1, arg2 ...
   Benchmark* Args(const std::vector<int>& args);
 
+  // Equivalent to Args({x, y})
+  // NOTE: This is a legacy C++03 interface provided for compatibility only.
+  //   New code should use 'Args'.
+  Benchmark* ArgPair(int x, int y) {
```

Comment:
I don't think so. Using the replacements requires C++11. So if you want C++03 benchmarks you still need this interface.